code/modules/clothing/suits/toggles.dm

```diff
@@ -59,7 +58,6 @@
 			else
 				H.equip_to_slot_if_possible(hood,slot_head,0,0,1)
 				suittoggled = 1
-				src.icon_state = "[initial(icon_state)]_t"
```

Comment:
The suit's icon changes when the hood is deployed. See the icons in icons/mob/suit.dmi (that you failed to remove despite removing the only place using those icons)